mesonbuild/interpreter/compiler.py

```diff
@@ -853,6 +860,7 @@ def get_argument_syntax_method(self, args: T.List['TYPE_var'], kwargs: 'TYPE_kwa
         KwargInfo('compile_args', ContainerTypeInfo(list, str), listify=True, default=[]),
         _INCLUDE_DIRS_KW,
         _DEPENDENCIES_KW.evolve(since='1.1.0'),
+        _DEPENDS_KW,
```

Comment:
```suggestion
        _DEPENDS_KW.evolve(since='1.4.0'),
```
So that trying to use this with an older minimum version will warn.